src/main/kotlin/io/gitlab/arturbosch/detekt/idea/DetektAnnotator.kt

```diff
@@ -75,5 +77,21 @@ class DetektAnnotator : ExternalAnnotator<PsiFile, List<Finding>>() {
         }
     }
 
+    private fun getSeverity(finding: Finding, hasCustomConfig: Boolean, treatAsError: Boolean): HighlightSeverity {
+        if (treatAsError) {
+            return HighlightSeverity.ERROR
+        }
+
+        if (!hasCustomConfig) {
+            return HighlightSeverity.WARNING
+        }
```

Comment:
Don't quite understand this clause? May I ask you to please elaborate what's the reason for it.

Reply:
I wanted to limit changes to the case when user does provide a custom detekt config.  WDYT?

Reply:
Yeah, I can see that. But with no provided custom config every issue is reported as **warning**. I don’t get why users would want that. We didn’t have this in previous versions.

Reply:
This is how the plugin behaves now so I'm just preserving current behavior.